packages/gasket-plugin-docs/README.md

```diff
@@ -67,10 +67,108 @@ as needed. The `defaults` are an available option to reference.
 #### Example setup
 
 ```js
+/**
```

Comment:
I'm not clear on why there are so many JS docs in this sample snippet.

Reply:
The models in a lot of the examples are unknown and possibly undocumented. As a user, new to Gasket, I could not(with relative ease) find any documentation on the return objects - what they do, are they required, what each property is responsible for. While most things you can somewhat infer their utility by name it's been a struggle to find a source of definitions.

Reply:
I did some preliminary work/experimenting on - In the docs plugin, parsing the `index.d.js` file content with `GasketConfig {}` and writing that to markdown. That initially was my direction for [PFX-81](https://jira.godaddy.com/browse/PFX-81) but after meeting with Andrew G it was decided that was more than the ticket required/intended. I think it could be very useful to parse the type files of installed plugins and writing a separate markdown file just for `gasket.config.js` options. 🤷‍♂️

Reply:
https://github.com/godaddy/gasket/blob/main/docs/typescript.md#validating-gasketconfigjs

Reply:
I'd recommend referencing the TypeScript types rather than duplicating all of these type definitions. Sure, it's possible someone doesn't have TypeScript installed in their project, but it's just sample code anyway, and it's weird to bloat it with all these docs.